TriblerGUI/widgets/trustpage/index.html

```diff
@@ -8,6 +8,7 @@
     <link rel="stylesheet" type="text/css" href="style.css">
 </head>
 <body>
+<button id="locate-me-button" onclick="locateMe()">Back To You</button>
```

Comment:
Please be consistent with the naming, also use back-to-you in id, function name etc.